BlueprintUICommonControls/Sources/ScrollView.swift

```diff
@@ -424,7 +424,7 @@ extension ScrollView {
             finalContentInset.bottom -= safeAreaInsets.bottom
         }
 
-        if #available(iOS 13, *) {
+        if #available(iOS 14, *) {
```

Comment:
[This radar](http://www.openradar.appspot.com/35866834) has been marked as resolved. It might be possible to remove this altogether, but this appears to be the behavior we're relying on at the moment so I did not wish to take that step as part of this PR.